get_matching_values/getmatchingvalue.js

```diff
@@ -0,0 +1,25 @@
+// receive 2 array. 
+// return matching value. if no match return null
```

Comment:
Or you'll read what I wrote in my comment again and change your code to match it.
If there's one or more matching values, return one of them (the first one is probably fine, but it is not specified in the question, so either one is fine). If there's no matches at all, return null.